examples/postgres-diskstats/dc-postgres.yml

```diff
@@ -0,0 +1,27 @@
+postgres_data:
+  image: juttle/postgres_diskstats_data
+
+postgres:
+  image: postgres
+  ports:
+    - 5432:5432
+
+outrigger_loader:
+# this container will load data into postgres via juttle, and exit.
+  image: juttle/outrigger:latest
+  links:
+    - postgres
+  volumes_from:
+    - postgres_data
+  volumes:
+    - ${PWD}/juttle-config.json:/opt/outrigger/.juttle-config.json
+    - ${PWD}/postgres-diskstats/loadfromscratch.sh:/config/loadfromscratch.sh
+  command: bash /config/loadfromscratch.sh
+
+outrigger:
+# this outrigger container will keep running to serve juttles.
+  links:
+    - postgres
+  volumes_from:
+    - postgres_data
```

Comment:
This is only to run programs by running juttle in the outrigger container, right?